### DISCUSSION TITLE
استفاده از سرور به عنوان پراکسی http

### DISCUSSION THREAD

**Discussion by alirezashamsaei** [hiddify/Hiddify-Manager#3294] [General سایر]
سلام، وقت به خیر و خسته نباشید به همه دست اندرکاران این پروژه بی‌نظیر.
میخواستم بدونم آیا امکانش هست که پنل این امکان رو برای کاربراش ایجاد میکنه که به صورت http proxy بهش متصل بشن؟
مثل اون جایی که توی bash ست میکنیم http_proxy
مثلا به این صورت:
http_proxy=http://USERNAME:PASSWORD@PROXY_HOST:PROXY_PORT

**Comment by CovERUshKA** [comment]
Any updates on this? This feature would be very useful.

Also, maybe this discussion would fit better in the "Ideas" category instead of "General".
